generators/app/templates/infra/gcp/cloudrun.tf

```diff
@@ -0,0 +1,24 @@
+# This will just create a default Cloud Run service but 
+# it will not update and mangae your revisions as Terraform is not meant to do this, that's a CD job.
```

Comment:
typo (mangae)

Reply:
We should specifically explain the behavior with the default image and the `ignore_changes`